build.rs

```diff
@@ -53,3 +58,19 @@ fn commit_date() -> Result<String, Some> {
         Err(Some {})
     }
 }
+
+fn docker_images() -> String {
+    let mut images = String::from("[");
+    let mut dir = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
+    dir.push("docker");
+    for entry in read_dir(dir).unwrap() {
+        let path = entry.unwrap().path();
+        if path.is_dir() {
```

Comment:
we should be using Path(Buf) here instead of depending on string concatenation. Much safer.

Reply:
Ah :+1: the path variable misled me.